package.json

```diff
@@ -207,7 +207,7 @@
     "stickyfill": "^1.1.1",
     "styled-components": "^3.4.5",
     "stylus": "^0.54.5",
-    "superagent": "^2.3.0",
+    "superagent": "^3.7.0",
```

Comment:
Would you mind checking to see if there are breaking changes across this version update?

Reply:
Yep! I checked on that during the upgrade:

> There are no big breaking changes and the upgrade should be smooth.

https://github.com/visionmedia/superagent/releases/tag/v3.0.0

Reply:
Sweet 👍